src/components/organisms/Settings/SettingPage/hooks.ts

```diff
@@ -55,33 +55,12 @@ export default (params: Params) => {
   const team = teams?.find(team => team.id === teamId);
```

Comment:
Just a question.
Can this line be removed? and remove `team` from `useEffect` dependencies.
I can't figure out it's purpose

Reply:
ohhh I'll double check but looks like maybe this was just a leftover from one of the useEffects I removed and didn't give me an error because it was still in the deps of the remaining one. Nice catch!